replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+fig3.table$gamble <- factor(fig3.table$gamble, levels = c("loss_gamble", "no_loss_gamble"))
+#relevel figure 3 table
+
+figure3 <- fig3.table %>%
+  ggplot(aes(x = title, y = values, fill = gamble)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("Attractiveness of gamble") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + ggtitle("Figure 3 - Attractiveness of loss or no-loss gamble\nfor Study 3\n(error bars represent 95% confidence interval)") + theme_bw() + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 3
+
+print(figure3)
+#displays figure 3
+```
+
+
+
+> The interaction between gamble and income was significant, F(1, 499) = 5.04, p < .05, ηp2 = .01.
+
+```{r echo = FALSE}
+# ANOVA : interaction effects
+summary(aov(judge ~ split * smallloss, df3_spss))
+temp_eta2 <- etaSquared(aov(judge ~ split * smallloss, df3_spss)) %>%
+  .[3,2] %>%
+  round(digit=2)
+print(c("partial eta squared:", temp_eta2), quote=F)
+```
+
+> The interaction was also significant when income was treated continuously, β = 1.13, t(499) = 2.21, p < .05.
+
+```{r echo = FALSE}
+# again, we find a larger beta coeffieicnt, but matching t value and p value.
+lm(scale(judge) ~ scale(logses) + scale(smallloss) + scale(logses * smallloss), data = df3_spss) %>%
+  tidy() %>%
+  filter(term == "scale(logses * smallloss)") %>%
+  mutate(beta = round(estimate, digit = 3),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+```
+
+---
+
+### Study 4
+
+> In Study 4, there were 263 participants (mean age = 31.6 years; 102 females, 161 males; median household size = 3 people; median household income = $45,000).
+
+```{r echo = FALSE}
+# Read data in
+suppressMessages(suppressWarnings(df4_1 <- read.csv("csv_files/4_Sheet1.csv", header=T)))
+suppressMessages(suppressWarnings(df4_2 <- read.csv("csv_files/4_Sheet2.csv", header=T)))
+suppressMessages(suppressWarnings(df4_4 <- read.csv("csv_files/4_Sheet4.csv", header=T)))
+suppressMessages(suppressWarnings(df4_spss <- read.spss("framing/study_4.sav", to.data.frame = TRUE)))
+```
+
+```{r echo = FALSE}
+# Number of subjects
+df4_2 %>%
+  select(Subject) %>%
+  summarise(n())
+
+# Mean age
+df4_2 %>%
+  summarise(mean(Age, na.rm=T)) %>%
+  round(digits=1)
+
+# Number of females vs. males
+df4_2 %>%
+  group_by(Gender) %>%
+  summarise(n())
+
+# Median household size
+df4_2 %>%
+  summarize(median(House))
+
+# Median household income
+df4_1 %>%
+  summarize(median(Income2))
+```
+
+> Four participants were excluded for not completing the prime question.
+
+```{r echo = FALSE}
+# find the four participants excluded
+df4_4 %>%
+  filter(Prime=="") %>%
+  select(Subject)
+# remove the 4 participants
+df4_1 <- df4_1 %>%
+  filter(Subject!=56, Subject!=75, Subject!=112, Subject!=236)
+
```

Comment:
I'm going to borrow this method for selecting out subjects by ID in the future! Thanks.